WordPress/metadata/PlayStoreStrings.po

```diff
@@ -0,0 +1,63 @@
+# Translation of Release Notes & Play Store Descriptions in English (US)
+# This file is distributed under the same license as the Release Notes & Play Store Descriptions package.
+msgid ""
+msgstr ""
+"PO-Revision-Date: 2018-05-23 11:49:24+0000\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: VsCode\n"
+"Project-Id-Version: Release Notes & Play Store Descriptions\n"
+
+msgctxt "release_note_101"
+msgid ""
+"10.1:\n"
```

Comment:
May I suggest that we add the 10.2 proposed release notes as well, just to begin with the latest we have:
```
10.2:
- New feature: Something there caught your eye? Read it any time with Save for later! Even when you're offline!
- Various bugfixes.
```